File: app/src/main/java/com/example/achalker/starbuzz/Food.java
package com.example.achalker.starbuzz;

public class Food {
    private String name;
    private String description;

    public  static final Food[] foods = {
        new Food("Croissant", "Light fluffy pastry"),
        new Food("Banana Nut Bread", "Walnut warm slice of bread"),
        new Food("Muffin", "Comfort food")
    };

    private Food (String name, String description) {
        this.name = name;
        this.description = description;
    }

    public String getDescription() {
        return description;
    }

    public String getName() {
        return name;
    }

    public String toString() {
        return this.name;
    }
}

File: app/src/main/java/com/example/achalker/starbuzz/model/Food.kt
package com.example.achalker.starbuzz.model

class Food private constructor(val name: String, val description: String) {
    companion object {
        @JvmField
        val foods = arrayOf(
                Food("Croissant", "Light fluffy pastry"),
                Food("Banana Nut Bread", "Walnut warm slice of bread"),
                Food("Muffin", "Comfort food")
        )
    }
}